content/en/docs/concepts/signals/traces.md

```diff
@@ -87,10 +87,17 @@ Sample Trace:
 }
 ```
 
-This sample trace output has three items, named "Hello-Greetings",
-"Hello-Salutations" and "Hello". Because each request's context has the same
-trace ID, all of the information can be tied together. This provides a trail
-through the requests' various routes, timestamps and other attributes.
+This sample trace output has three distinct log-like items, called
+[Spans](#spans-in-opentelemetry), named `Hello-Greetings`, `Hello-Salutations`
+and `Hello`. Because each request's context has the same `trace_id`, they are
+considered to be a part of the same Trace.
+
+Another think you'll note is that each Span of this example Trace looks like a
+structured log. That's because it kind of is! One way to think of Traces is that
+they're a collection of structured logs with context, correlation, heirarchy,
```

Comment:
```suggestion
they're a collection of structured logs with context, correlation, hierarchy,
```

glas I am not the only one making this typo every freaking time 😆

Reply:
I'm half german, so therefore I am doomed to do this too